### PR TITLE
Add ResourceGetHandler endpoint

### PR DESCRIPTION
**Pull request by damnski**
ResourceGetHandler endpoint gets arbitrary resource costs within a space

### CODE REVIEW

File: api/handlers_resourcenames.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
+package api
```
Comment (fishnix) [inline api/handlers_resourcenames.go:1]: This is still part of the space handlers IMO, I don't think it warrants a new file.

File: api/handlers_resourcenames.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/costexplorer"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+
+	ce "github.com/YaleSpinup/cost-api/costexplorer"
+	log "github.com/sirupsen/logrus"
+)
+
+// ResourceGetHandler gets the cost for a space, grouped by the service.  By default,
```
Comment (fishnix) [inline api/handlers_resourcenames.go:19]: `SpaceResourceGetHandler` probably

Comment (fishnix) [reply api/handlers_resourcenames.go:19]: I think the comment is wrong?

File: api/handlers_resourcenames.go [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+			handleError(w, err)
+			return
+		}
+	}
+
+	/*
+		https://docs.aws.amazon.com/aws-cost-management/latest/APIReference/API_GetCostAndUsage.html
+
+		https://docs.aws.amazon.com/sdk-for-go/api/service/costexplorer/
+	*/
```
Comment (fishnix) [inline api/handlers_resourcenames.go:77]: Use the `//` style comments

**Review by fishnix — COMMENTED** [review]
Looks good - deploy a prerelease to test?  I'll poke it there.

File: api/handlers_resourcenames.go (hunk 1 above)
Comment (damnski) [reply api/handlers_resourcenames.go]: :+1:

**Comment by fishnix** [comment]
this seems like it works, its hard to tell but I had some older resources that returned different number for different date ranges.  👍   also this should be a minor version, not a patch

**Pull request merged**